back-office/shop-parameters/product-settings/product-settings-page.md

```diff
@@ -0,0 +1,146 @@
+
+**Display available quantities on the product page.** By default, it is enabled, **displaying in the front office how many items are in stock** in the 'Product Details' section of the product page. Number of available items should be based on the selected attributes (= combination).
+
+**Display remaining quantities when the quantity is lower than.** Merchants can **display a small alert in the front office when the stock for a product gets below a certain level**. It adds the following message, _Last items in stock_, right below the 'Add to cart' button in the product page. By default, it is set to 3 - typing 0 disables this feature.
+
+Only numbers can be typed in the field, or added by using the arrows at the end of the field. It cannot be set to less than 0, otherwise the field goes back to the previous valid value and an error notification is displayed: _The Display remaining quantities when the quantity is lower than field is invalid._
+
+
+_As a merchant, I want to be able to manage my unavailable product combinations._
+
+**Display unavailable attributes on the product page.** By default, it is enabled. And the tooltip should be _If an attribute is not available in every product combination, it will not be displayed._, cf. _1.7.8_ issue #[17447](https://github.com/PrestaShop/PrestaShop/issues/17447).
+
+**When this option is enabled, unavailable products' combinations remain visible in the shop**. A message on the FO product page indicates the product is no longer available in the chosen option and invites customers to choose another combination: _Product available with different options_. 
+
+**When this option is enabled as well as the 'Allow ordering out-of-stock products' option**, customers are able to add unavailable combinations to their cart.
+ 
+**When this option is disabled and if the products have one single attribute**, their combinations are no longer available and the attribute is hidden.
+ 
+For example, the XS attribute and the related product are not displayed in the following case:
+
+| Size  |  In stock |
+|---|---|
+| XS  | No  |
+| S  | Yes  |
+| M  | Yes  |
+| L  | Yes  |
+
+but the S, M, and L attributes remain displayed.
+ 
+Also, **when this option is disabled and if the value of an attribute has no product in all possible combinations**, the combination of the products that are no longer available is hidden.
+ 
+For example, the white attribute and the related product are not displayed in the following case:
+
+| Size  | Color | In Stock |
+|---|---|---|
+| S  | White  | No |
+| S | Black | Yes  |
+| M | White | No |
+| M | Black | Yes |
+
+but the S, M, and black attributes remain displayed.
+
+
+_As a merchant, I want to be able to parameter my products' attributes._
+
+**Display the "Add to cart" button when a product has attributes.** By default, it is enabled. Display or hide the 'Add to cart' button on category pages for products that have attributes forcing customers to see product details.
```

Comment:
It's not working when enabled with the default theme, the add to cart button has been removed from the FO products lists in 1.7.0 with the default theme because a lot of users were adding to the cart a product without choosing the combination
But maybe it's working with other themes. Maybe we could add a tooltip or message under this toggle, wdyt ?

Reply:
Right, I wanted to ask you about this feature as I could not manage to reproduce the behavior in the FO. Now I know why, thanks a lot! But if it is a wanted behavior, why is it still in the BO? BC if it is handled by other themes?

Reply:
Yes I think it works with other themes and that's why the option remains in the BO

Reply:
How intuitive! 🙄

@Julievrz, what do you think about informing the user about it? Should it be part of the current tooltip or be displayed as a help text under the switch button?

Reply:
I think it would be better to display a help text under the switch button. 

"This feature cannot be enabled on stores using the default theme." in Admin.Shopparameters.Help 

Wdyt?